test/parallel/test-zlib-failed-init.js

```diff
@@ -5,6 +5,10 @@ require('../common');
 const assert = require('assert');
 const zlib = require('zlib');
 
+if (process.config.variables.node_shared_zlib && /^1\.2\.[0-8]$/.test(process.versions.zlib)) {
+  require('../common').skip("older versions of shared zlib don't throw on create");
```

Comment:
`../common` is already required above. You just need to assign it to a variable.